src/main.rs

```diff
@@ -68,12 +71,12 @@ fn app() -> Result {
     let mut cmds = Commands::new();
 
     // Tags
-    cmds.add("?tag {key}", tags::get);
```

Comment:
Why change this to plural? It's for looking up a single key. While inconsistent with the rest of the commands, I think it makes sense to be an outlier.